core/translate/insert.rs

```diff
@@ -332,6 +320,43 @@ pub fn translate_insert(
             dest_reg: record_reg,
         });
 
+        let make_record_label = program.allocate_label();
```

Comment:
The earlier `make_record_label` (above this for loop) should now be renamed because it's not pointing to a `MakeRecord` instruction anymore